packages/local_auth/local_auth_darwin/lib/types/auth_messages_macos.dart

```diff
@@ -0,0 +1,107 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:intl/intl.dart';
+import 'package:local_auth_platform_interface/types/auth_messages.dart';
+
+/// Class wrapping all authentication messages needed on macOS.
+/// Provides default values for all messages.
+@immutable
+class MacOSAuthMessages extends AuthMessages {
+  /// Constructs a new instance.
+  const MacOSAuthMessages({
+    this.lockOut,
+    this.goToSettingsButton,
```

Comment:
This is no longer used.

Reply:
Sending a string from Dart to the host isn't using the string in a meaningful sense, it's just passing it around. The reason these strings exist is to provide them to native code in order to display them to the user in a native dialog. There is no longer any code path that results in this string being displayed to the user, so it's dead code.

Reply:
If I remove MacOSAuthMessages, should I rename iOSAuthMessages to DarwinAuthMessages?

Reply:
> If I remove MacOSAuthMessages

You should not remove `MacOSAuthMessages`. You should remove the parts of it that are never used.

> should I rename iOSAuthMessages to DarwinAuthMessages?

No; as discussed [here](https://github.com/flutter/packages/pull/6267#discussion_r1546793569) they are not the same. That would also re-create a different version of the same problem: a public API surface that allows setting some values for macOS that will never be used on macOS.